.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -7,17 +7,71 @@ assignees: ''
 
 ---
 
+- [ ] Check for critical performance regressions.
 - [ ] Check [Nightlies](http://cdash.seqan.de/index.php?project=SeqAn3) for critical build failures.
-- [ ] Check Changelog.md for completeness.
-- [ ] Prepare `seqan-[VERSION]-with-submodules.tar.gz`.
-- [ ] Update the index from cppreferences.com so that up-to-date documentation links are generated.
+- [ ] Check the [Changelog.md](https://github.com/seqan/seqan3/blob/master/CHANGELOG.md) for completeness.
+- [ ] Update the index from cppreference.com so that up-to-date documentation links are generated.
+  <details><summary>Click for detailed steps</summary><br>
+
+  Check for [new releases](https://github.com/PeterFeicht/cppreference-doc/releases) and update the link and hash
+  in the [code base](https://github.com/seqan/seqan3/blob/b0b279689fa65c2431a5162f2d8acc3ca663f72d/test/documentation/seqan3-doxygen.cmake#L37).
+
+  For the hash do
+
+  ```
+  wget -O- <link to html book> | sha256sum
+  ```
+
+  </details>
 - [ ] Add versioned documentation build on docs.seqan.de.
+  <details><summary>Click for detailed steps</summary>
```

Comment:
I think these steps must be done after the commit has  been tagged.
Then the documentation and the submodule tar should be build from that commit. 
Doing it afterwards increases the risk of unwanted differences because something urgently had to be merged inbetween someone preparing these artefacts and someone tagging the release.

Reply:
For the documentation, I agree, since it does not really matter (I just wanted to make as many things beforehand as possible).
But for the `prepare seqan-[VERSION]-with-submodules.tar.gz` I think it is important to attach it directly when tagging the release because
1) package managers like Bioconda might automatically detect new releases and try to pull the files before we can attach them by editing the release afterwards.
2) If we move to automatic deployment, we will very probably also prepare this beforehand and attach it directly no?

And I mean these steps take 5 minutes and if you prepare the release you freeze master anyway?

Reply:
1. I don't think that downstream package manager automatically pull from our repository. At least I am not aware of this. That would be totally awesome. 
2. The point is that the workflow, if you would automate it, would be to first tag and then upload the assets. How, can you avoid this latency. So preparing the asset from the tagged commit is in my opinion the only safe thing to do.

>And I mean these steps take 5 minutes and if you prepare the release you freeze master anyway?

First, they take also after tagging only 5 minutes. Second, these things can already be automated using existing GitHub actions.

Reply:
> 1. I don't think that downstream package manager automatically pull from our repository. At least I am not aware of this. That would be totally awesome.

They do, so it would be nice if the release is buildable... 
https://github.com/bioconda/bioconda-recipes/pull/19893